sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample11_ClassifyCustomCategories.md

```diff
@@ -0,0 +1,99 @@
+
+```C# Snippet:CreateTextAnalyticsClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var client = new TextAnalyticsClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+```
+
+## Performing Custom Multiple Category Classification in one or multiple documents
+
+To perform Multiple Category Classification in one or multiple documents, set up a `MultiCategoryClassifyAction` and call `StartAnalyzeActionsAsync` on the documents. The result is a Long Running Operation of type `AnalyzeActionsOperation` which polls for the results from the API.
+
+```C# Snippet:TextAnalyticsMultiCategoryClassifyAsync
+// Get input document.
+string document = @"I need a reservation for an indoor restaurant in China. Please don't stop the music. Play music and add it to my playlist.";
+
+// Prepare analyze operation input. You can add multiple documents to this list and perform the same
+// operation to all of them.
+var batchInput = new List<string>
+{
+    document
+};
+
+// Set project and deployment names of the target model
+string projectName = "<projectName>";
+string deploymentName = "<deploymentName>";
+
+var multiCategoryClassifyAction = new MultiCategoryClassifyAction(projectName, deploymentName);
+
+TextAnalyticsActions actions = new TextAnalyticsActions()
+{
+    MultiCategoryClassifyActions = new List<MultiCategoryClassifyAction>() { multiCategoryClassifyAction }
+};
+
+// Start analysis process.
+AnalyzeActionsOperation operation = await client.StartAnalyzeActionsAsync(batchInput, actions);
+
+await operation.WaitForCompletionAsync();
+```
+
+The returned `AnalyzeActionsOperation` contains general information about the status of the operation. It can be requested while the operation is running or when it has completed. For example:
+
+```C# Snippet:TextAnalyticsMultiCategoryClassifyOperationStatus
+// View operation status.
+Console.WriteLine($"AnalyzeActions operation has completed");
+Console.WriteLine();
+
+Console.WriteLine($"Created On   : {operation.CreatedOn}");
+Console.WriteLine($"Expires On   : {operation.ExpiresOn}");
+Console.WriteLine($"Id           : {operation.Id}");
+Console.WriteLine($"Status       : {operation.Status}");
+Console.WriteLine($"Last Modified: {operation.LastModified}");
+Console.WriteLine();
+```
+
+To view the final results of the long-running operation:
+
+```C# Snippet:TextAnalyticsMultiCategoryClassifyAsyncViewResults
+// View operation results.
+await foreach (AnalyzeActionsResult documentsInPage in operation.Value)
+{
+    IReadOnlyCollection<MultiCategoryClassifyActionResult> multiClassificationActionResults = documentsInPage.MultiCategoryClassifyResults;
+
+    foreach (MultiCategoryClassifyActionResult classificationActionResults in multiClassificationActionResults)
+    {
+        foreach (MultiCategoryClassifyResult documentResults in classificationActionResults.DocumentsResults)
+        {
+            if (documentResults.ClassificationCategories.Count > 0)
```

Comment:
I thought arch board feedback was to rename this property to `classifications`? @mssfang

Reply:
Thanks for pointing that out. I seem to have missed this rename in `SingleCategoryClassifyResult` as well. 

`SingleCategoryClassifyResult.ClassificationCategory` -> `SingleCategoryClassifyResult.Classification`

`MultiCategoryClassifyResult.ClassificationCategories` -> `MultiCategoryClassifyResult.Classifications`